bin/MorphingMSSMvsSM.cpp

```diff
@@ -283,7 +283,7 @@ int main(int argc, char **argv) {
   // Define categories
   map<string, Categories> cats;
   // STXS stage 0 categories (optimized on ggH and VBF)
-  if(analysis == "mssm_vs_sm_classic" || analysis == "mssm_vs_sm_heavy" || analysis == "mssm_vs_sm_classic_h125" || analysis == "mssm_vs_sm_CPV"){
```

Comment:
Doesn't this remove the categorisation for the CPV model-dependent limits entirely from the datacards?

Reply:
I think it could make sense to leave the mssm_vs_sm_classic analysis in the morphing since I think it is useful to see the effect of the sm categorisation for the model-dependent limits.

Reply:
> I think it could make sense to leave the mssm_vs_sm_classic analysis in the morphing since I think it is useful to see the effect of the sm categorisation for the model-dependent limits.

i fully agree 



> Doesn't this remove the categorisation for the CPV model-dependent limits entirely from the datacards?

Which categorisation do we want to use for the CP scenario ? Do we also need to versions aka 
`mssm_vs_sm_CPV` and `mssm_vs_sm_CPV_classic` ?

Reply:
For now I guess I would leave it as it is, so without classic in the name but only classic categories because this is the setup that should currently work. I haven't tried it out so far, so I don't really know if it is easily extendable to the SM categories without surprises. When we get back to work more on this scenario we can still add the SM categories in my opinion.